hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestReplicationSource.java

```diff
@@ -662,12 +666,16 @@ public void testAgeOfOldestWal() throws Exception {
       manualEdge.setValue(10);
       // Diff of current time (10) and  log-walgroup-a.8 timestamp will be 2.
       source.enqueueLog(log1);
+      // Sleep for chore to update WAL age
+      Thread.sleep(1000);
```

Comment:
Use Waiter.waitFor method to wait on some condition. That way if the jenkins machine are slow, it will not create flaky tests.